packages/shared/lib/services/environment.service.ts

```diff
@@ -136,7 +136,7 @@ class EnvironmentService {
             .select<Account>('_nango_accounts.*')
             .from(TABLE)
             .join('_nango_accounts', '_nango_accounts.id', '_nango_environments.account_id')
-            .where({ id: environment_id })
+            .where('_nango_environments.id', environment_id)
```

Comment:
@bodinsamuel noticed this happening in APM often 
![image](https://github.com/NangoHQ/nango/assets/1724137/df42b462-8860-49ad-bdee-428dc11f5ffd)

Reply:
arg nice catch thanks